docs/spec/channel/version_060.md

```diff
@@ -0,0 +1,137 @@
+# Channel Spec (Version 0.6.0)
+
+## Background
+
+The current system consists of two major APIs, the `Channel` and their `ClusterChannelProvisioner`. Both are living in the `eventing.knative.dev/v1alpha1` API Group.
+
+### The Channel
+
+A `kind: Channel` logically receives events on its input domain and forwards them to its subscribers. Below is a specification for the different parts of the `kind: Channel`.
+
+### The ClusterChannelProvisioner
+
+Describes an abstract configuration of a Source system which produces events or a Channel system that receives and delivers events.
```

Comment:
Maybe clarify what the provisioner does - it's not just descriptive, it creates new channel instances does it not?